### ISSUE TITLE
custom letter does a bad (Ñ)

### ISSUE THREAD

**Issue by Chonono** [krzys-h/UndertaleModTool#679]
it does a bad when i try to translate
![image](https://user-images.githubusercontent.com/93895408/140679324-681c6296-2fb7-4eeb-b100-1b855e4a1232.png)

I've tried this thing for like 3 hours now, it's really hard. Yeah, I've looked at issue 8, but it doesn't really work for me since I don't have GMS so I tried the first thing and now every custom letter is, like, really high up compared to the other ones.

Any ideas why this might be the case?

**Comment by SlayerIbn** [comment]
hay traducciones que tienen de forma perfecta este tipo de letras, incluso la mia tambien las tiene bien jeje

**Comment by Chonono** [comment]
I fixed it by making the letters start 3 pixels taller than the first pixel, somehow making it land perfectly at the base of the other letters. Also, yes I know that there are more translations that fix it, but I'm making one myself for a shitpost where it uses latinamerican jokes.

TL;DR If you have the same issue, try making the "source" pixel higher than the actual first pixel of the letter is.
